### PR TITLE
Added pass completion

### PR DESCRIPTION
**Pull request by xFA25E**

### CODE REVIEW

File: pcmpl-args.el [hunk 1]
```diff
@@ -3389,6 +3390,136 @@ options found in its man page."
...
+       (option "-q[LINENUMBER], --qrcode[=LINENUMBER]" (("LINENUMBER" none))
+               :help "Display a QR code of the first (or specified) line")
+       (argument 0 (("PASSNAME" (:eval (pcmpl-args-pass-find "-type" "f")))))))
+
+    ((or "cp" "mv")
+     '((option "-f, --force" :help "Silently overwrite NEWPATH if it exists")
+       (argument 0 (("OLDPATH" (:eval (pcmpl-args-pass-find)))))
+       (argument 1 (("NEWPATH" (:eval (pcmpl-args-pass-find)))))))))
+
+(defun pcomplete/pass ()
```
Comment (CeleritasCelery) [inline pcmpl-args.el:3532]: This function appears unused

File: pcmpl-args.el [hunk 2]
```diff
@@ -3389,6 +3390,136 @@ options found in its man page."
...
+     ("help" "Show usage message")
+     ("init" "Initialize new password storage")
+     ("insert" "Insert a new password into the password store")
+     ("ls" "List names of passwords")
+     ("mv" "Move password or directory")
+     ("rm" "Remove password or directory")
+     ("show" "Decrypt and print a password")
+     ("version" "Show version information"))))
+
+(defun pcmpl-args-pass-prefix ()
```
Comment (CeleritasCelery) [inline pcmpl-args.el:3423]: Let's add docstrings to these newly created functions

File: pcmpl-args.el [hunk 3]
```diff
@@ -3389,6 +3390,136 @@ options found in its man page."
...
+        (while (search-forward-regexp rx nil t)
+          (replace-match "\\1")))
+      (forward-line 1)
+      (let (lines)
+        (while (progn (forward-line -1)
+                      (push (string-trim-right (thing-at-point 'line t)) lines)
+                      (not (bobp))))
+        (cl-set-difference lines (cadr (assq '* args)) :test #'string=)))))
+
+(defun pcmpl-args-pass-command-specs (cmd)
```
Comment (CeleritasCelery) [inline pcmpl-args.el]: Can you add a new section to the end of the README explaining how to define new pass-through completions?

**Comment by xFA25E** [comment]
Thanks for the feedback. Also, I forgot to add `pcomplete/pass` to autoloads.

File: pcmpl-args.el (hunk 3 above)
Comment (xFA25E) [reply pcmpl-args.el]: What do you mean by "pass-through" completions?

**Comment by xFA25E** [comment]
Tell me if you see other problems here. 

For "pass-through" completions explanation should I do a separate PR? Can you explain in more details what should I do?

Thanks!

**Comment by CeleritasCelery** [comment]
> For "pass-through" completions explanation should I do a separate PR? Can you explain in more details what should I do?

Maybe I misunderstood this PR. You originally opened #7, about having sub-commands and "passing though" the completions to the `pass` command. I assumed that this PR was trying to implement some of that sub-command completion. But it looks like this is not the case? you are just adding completion for the `pass` command correct?

**Comment by xFA25E** [comment]
Correct, just the completion for `pass`. I'm going to send PRs for parted and, maybe, systemctl soon.  After a bit of testing

**Comment by xFA25E** [comment]
Originally, I asked that question about subcommands, because I didn't understand how the library worked. I was just copy-pasting parts of the source code. But, now, I know how it works. It is true that the library lacks documentation, though. In the docstrings, it is literally written "go read source code to understand it" :)

**Comment by CeleritasCelery** [comment]
Thanks!

**Pull request merged**